crates/containerd-shim-wasmtime/src/main.rs

```diff
@@ -1,6 +1,7 @@
-use containerd_shim_wasm::sandbox::cli::{revision, shim_main, version};
+use containerd_shim_wasm::sandbox::cli::{revision, shim_main_with_otel, version};
 use containerd_shim_wasmtime::WasmtimeInstance;
 
-fn main() {
-    shim_main::<WasmtimeInstance>("wasmtime", version!(), revision!(), "v1", None);
+#[tokio::main]
```

Comment:
If we create the tokio runtime manually in `shim_main_with_otel`, then we woulnd't need to impose tokio as a dependency for runtime crates.
Also it's a bit odd that we are calling a sync/blocking function inside an async function. IIUC, this would break if we use `#[tokio::main(flavor = "current_thread")]` as we would never yield back to the tokio runtime.

Reply:
I've moved tokio runtime creation inside `shim_main_with_otel`.